content/talks/django-end-to-end-testing.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Effective end-to-end testing for Django
+date: 2022-10-16
+author: ed
+image:
+  src: talks/computers.jpg
+  alt: |
+    People gathered around a table with multiple computers, cellphones, and
+    notebooks on top.
+  position: top
+tags:
+  - Testing
+  - Tutorial
+  - Django
+  - Python
+slides: https://drive.google.com/file/d/1K3Fyn4uj2uZoJSO5H5mFCMXr6206kiNa/views
+events:
+  - venue: DjangoCon US 2022
+    adr: San Diego, CA (hybrid)
+    url: https://2022.djangocon.us/tutorials/effective-end-to-end-testing-for-django/
+    date: 2022-10-16
+    end: 2022-10-21
+    slug: 2022-djangocon
```

Comment:
I'm not sure what this `slug` field does. Let me know if it's wrong 😅

Reply:
I think this is only used for press quotes, not events.

```suggestion
```